exercises/pythagorean-triplet/PythagoreanTriplet.fs

```diff
@@ -4,4 +4,4 @@ let triplet x y z = failwith "You need to implement this function."
 
 let isPythagorean triplet = failwith "You need to implement this function."
```

Comment:
This function does not need to be implemented anymore, and can thus be removed.

Reply:
OK, done.